tools/interop_matrix/client_matrix.py

```diff
@@ -293,3 +293,21 @@ def should_build_docker_interop_image_from_release_tag(lang):
         },
     ],
 }
+
+# This matrix lists the version of testcases to use for a release. As new
```

Comment:
Any good reason why isn't this data part of the LANG_RELEASE_MATRIX  data structure? It seems to me that this is exactly the data that belongs in it - so I don't understand why we are introducing an adhoc data structure TESTCASES_VERSION_MATRIX.

Reply:
My first version is actually what you suggested but I thought the release matrix is about the version of the libraries themselves, not tests. But if you prefer this way, I can change it back.

Reply:
Plus I found things become messier for csharp if I just add this to LANG_RELEASE_MATRIX, because test versions are different between csharp and csharpcoreclr. With a separate dictionary, I can simply do something like this:
'csharp_v1.1.4': 'csharp__v1.1.4',
'csharpcoreclr_v1.15.0': 'csharpcoreclr__v1.10.1'. (This is hypothetical, just an example)

Reply:
Thanks for explanation. Leaving the resolution up to you.

Reply:
Thanks for the review!